### DISCUSSION TITLE
Minimum .NET version

### DISCUSSION THREAD

**Discussion by daflh** [SafeExamBrowser/seb-win-refactoring#368] [Q&A] [answered]
What is the minimum .NET Framework version for SEB 3.3.2 to works? Is it 4.7.2?

**Comment by dbuechel** [comment, marked as answer]
Yes, the .NET Framework 4.7.2 Runtime is sufficient for version 3.3.2: https://dotnet.microsoft.com/download/dotnet-framework/net472. All required runtime dependencies are always listed in the release notes, see https://github.com/SafeExamBrowser/seb-win-refactoring/releases/tag/3.3.2.

**Reply by daflh** [reply]
So .NET 4.6 doesn't work? Because it looks like Windows 10 version 1511 doesn't support .NET version above 4.6.x (including 4.7.2).

**Reply by dbuechel** [reply]
> So .NET 4.6 doesn't work?

Exactly, you'll need at least version 4.7.2.

> Because it looks like Windows 10 version 1511 doesn't support .NET version above 4.6.x

We'd recommend you to update your Windows, just for security reasons alone it'd be worth to update to the latest version.

**Reply by daflh** [reply]
All right, thanks
